Kiosk/IntelligentKioskSample.csproj

```diff
@@ -22,11 +22,15 @@
     <SccProvider>SAK</SccProvider>
     <AppxAutoIncrementPackageRevision>True</AppxAutoIncrementPackageRevision>
     <AppxBundle>Always</AppxBundle>
-    <AppxBundlePlatforms>x86|x64|arm</AppxBundlePlatforms>
```

Comment:
these next few lines probably shouldn't be changing with your code additions.  can you revert these?

Reply:
I can definitely revert those back to their original state. These ended up changing when I was creating app bundles for demoing purposes.